eu.ngdevelopment.tsukitag.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "eu.ngdevelopment.tsukitag",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "TsukiTag",
+    "finish-args": [
+        "--filesystem=xdg-pictures",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

Comment:
does it support wayland?

Also `--share=ipc` is needed for x11 anyway.

Reply:
Yes, Wayland was tested with a Fedora vm installation from the latest ISO.